src/audio5.js

```diff
@@ -941,7 +954,7 @@
      */
     play: function () {
       if(!this.playing){
-        this.audio.play();
+        return this.audio.play();
```

Comment:
Only changed in /src/audio5 as it looks like previous change was made to build output rather than src